neurodamus/cell_distributor.py

```diff
@@ -201,11 +201,15 @@ def load_nodes(self, load_balancer=None, *, _loader=None, loader_opts=None):
         loader_f = (lambda *args: _loader(*args, **loader_opts)) if loader_opts else _loader
 
         logging.info("Reading Nodes (METype) info from '%s'", conf.CellLibraryFile)
-        if load_balancer and load_balancer.population != self._target_spec.population:
+        if load_balancer and \
```

Comment:
For improving the code now I'd create a `MemoryLoadBalance` class that offers a similar API to the old load balance, namely have the `.population` property and related methods, like `get_local_gids(rank)` `valid_load_distribution()`.